nixos/lib/make-bootloader-options.nix

```diff
@@ -0,0 +1,35 @@
+{ lib }:
+/* Build a bootloader set of options */
+let
+  inherit (lib) mkOption types mdDoc;
+
+  commonOptions = {
+      id = mkOption {
```

Comment:
Indentation is weird. Please run `nixpkgs-fmt` or the tool in the recent formatting RFC.